src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,24 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.tasklist.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Encapsulates a command given to Apollo.
+ *
+ * @author Kartikeya
+ */
+public interface Command {
+    /**
+     * Executes the command action.
+     *
+     * @param itemList list of tasks
+     * @param ui       ui interactions with user
```

Comment:
Maybe removing the empty spaces between ui and ui would be neater and more standardised.

Reply:
I was unsure of the proper format to follow here myself. I resorted to following the [Oracle standard](https://www.oracle.com/sg/technical-resources/articles/java/javadoc-tool.html) linked on the CS2103/T website, which mentioned 
> Additional spaces can be inserted between the name and description so that the descriptions line up in a block.
As such, given that I've standardized it across all my code I'll stick to this.